docs/install/eks.md

```diff
@@ -0,0 +1,158 @@
+
+This guide shows you how to deploy Percona Operator for MongoDB on Amazon
+Elastic Kubernetes Service (EKS). The document assumes some experience with the
+platform. For more information on the EKS, see the [Amazon EKS official documentation](https://aws.amazon.com/eks/).
+
+## Prerequisites
+
+Before creating an EKS cluster, make sure to:
+
+1. [Install the AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for interacting with the different parts of AWS.
+
+2. [Install eksctl](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) to simplify cluster creation on EKS.
+
+3. [Install kubectl](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) to manage and deploy applications on Kubernetes.
+
+4. [Configure AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) with your credentials.
+
+## Create the EKS cluster
+
+1. Settled the required cluster details:
+
+    * name of your EKS cluster
+    * AWS region in which you wish to deploy your cluster
+    * the number of nodes you would like to have
+    * the desired ratio between [on-demand](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html)
+        and [spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) instances in the total number of nodes.
+
+    !!! note
+
+        [spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) instances are not recommended for production environment but may be useful for testing purposes.
+
+2. Create your EKS cluster [following the official cluster creation instructions](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+3. [Install the Amazon EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) on your cluster. See the [official documentation](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html) on adding it as an Amazon EKS add-on.
+
+## Install the Operator and deploy your MongoDB cluster
+
+1. Deploy the Operator. By default deployment will be done in the `default`
+    namespace. If that's not the desired one, you can create a new namespace
+    and/or set the context for the namespace as follows (replace the `<namespace name>` placeholder with some descriptive name):
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl create namespace <namespace name>
+    $ kubectl config set-context $(kubectl config current-context) --namespace=<namespace name>
+    ```
+
+    At success, you will see the message that `namespace/<namespace name>` was created, and the context was modified.
+
+    To deploy the Operator [use](https://kubernetes.io/docs/reference/using-api/server-side-apply/) the following command:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/bundle.yaml
```

Comment:
We need recommending provision everest

Reply:
Removed everything following the Create the EKS cluster section